Java/core/src/main/java/com/amazon/randomcutforest/tree/Node.java

```diff
@@ -422,4 +422,24 @@ protected void deleteSequenceIndex(long sequenceIndex) {
             sequenceIndexes.remove(sequenceIndex);
         }
     }
+
+    /**
+     * The following function returns the bounding box corresponding to the child
+     * that is not the path from the node to leaf traversed by a point. If such a
+     * box cannot be defined then the function returns null.
+     * 
+     * @param point the point as above
+     * @param node  the node as above
+     * @return the bounding box of the node
+     */
+
+    public static BoundingBox getSiblingBoundingBox(double[] point, Node node) {
```

Comment:
hIt doesn't look the the `point` argument is being used.

Reply:
isLeftOf uses the argument "point" ... that may be a lengthier change ..